qa/workunits/fs/snaps/snaptest-name-limits.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# This tests snapshot names limits: names have to be < 240 chars
+#
+
+function cleanup ()
+{
+	rmdir d1/.snap/*
```

Comment:
drop this, as d1 is removed entirely in the next line?

Reply:
That would be true for regular directories. `.snap` is special and you can't remove `d1` if you still have snapshots in it.  (See also comment bellow.)